Sources/SignalProducer.swift

```diff
@@ -217,6 +360,39 @@ extension SignalProducer: SignalProducerProtocol {
 }
 
 extension SignalProducerProtocol {
+	/// Lift an unary Signal operator to operate upon SignalProducers instead.
+	///
+	/// In other words, this will create a new `SignalProducer` which will apply
+	/// the given `Signal` operator to _every_ created `Signal`, just as if the
+	/// operator had been applied to each `Signal` yielded from `start()`.
+	///
+	/// - parameters:
+	///   - transform: An unary operator to lift.
+	///
+	/// - returns: A signal producer that applies signal's operator to every
+	///            created signal.
+	public func lift<U, F>(_ transform: @escaping (Signal<Value, Error>) -> Signal<U, F>) -> SignalProducer<U, F> {
```

Comment:
These trampolines are made as `liftLeft` and `liftRight` cannot be included in `SignalProducerProtocol`, and `lift` is here for consistency. Having said that, these trampolines can be removed when Swift 3.1 lands, should we drop the protocol in favour of concrete same-type requirements.